docs/quickstart/create-and-publish-a-package-using-visual-studio.md

```diff
@@ -1,162 +1,188 @@
+
+- [Register for a free account on nuget.org](../nuget-org/individual-accounts.md#add-a-new-individual-account) if you don't have one already. You must register and confirm the account before you can upload a NuGet package.
+
+- Install the NuGet CLI by downloading it from [nuget.org](https://dist.nuget.org/win-x86-commandline/latest/nuget.exe). Add the *nuget.exe* file to a suitable folder, and add that folder to your PATH environment variable.
 
 ## Create a class library project
 
-You can use an existing .NET Standard Class Library project for the code you want to package, or create a simple one as follows:
+You can use an existing .NET Standard Class Library project for the code you want to package, or create one as follows:
 
-1. In Visual Studio, choose **File > New > Project**, expand the **Visual C# > .NET Standard** node, select the "Class Library (.NET Standard)" template, name the project AppLogger, and click **OK**.
+1. In Visual Studio, select **File** > **New** > **Project**.
 
-   > [!Tip]
-   > Unless you have a reason to choose otherwise, .NET Standard is the preferred target for NuGet packages, as it provides compatibility with the widest range of consuming projects.
+1. In the **Create a new project** window, select **C#**, **Windows**, and **Library** in the dropdown lists.
 
-1. Right-click on the resulting project file and select **Build** to make sure the project was created properly. The DLL is found within the Debug folder (or Release if you build that configuration instead).
+1. In the resulting list of project templates, select **Class Library** (with the description, *A project for creating a class library that targets .NET or .NET standard*), and then select **Next**.
 
-Within a real NuGet package, of course, you implement many useful features with which others can build applications. For this walkthrough, however, you won't write any additional code because a class library from the template is sufficient to create a package. Still, if you'd like some functional code for the package, use the following:
+1. In the **Configure your new project** window, enter *AppLogger* for the **Project name**, and then select **Next**.
 
-```cs
-namespace AppLogger
-{
-    public class Logger
-    {
-        public void Log(string text)
-        {
-            Console.WriteLine(text);
-        }
-    }
-}
-```
+1. In the **Additional information** window, select **.NET Standard 2.1** (or the latest version) for **Framework**, and then select **Create**.
+
+   > [!TIP]
+   > Unless you have a reason to choose otherwise, .NET Standard is the preferred target for NuGet packages, as it provides compatibility with the widest range of consuming projects. For more information, see [Create and publish a package using Visual Studio (.NET Standard)](create-and-publish-a-package-using-visual-studio.md).
```

Comment:
I don't think that .NET Standard is still the preferred target. .NET 5+ stopped using .NET Standard and the "no platform" TFM (net6.0) is the new cross-platform library, whereas platform specific TFMs (net6.0-windows) extend the base set of APIs.

@terrajobst do you agree that we should no longer claim that .NET Standard is preferred?


I didn't read this carefully, but this is relevant: https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-1-0#when-to-target-net50-or-net60-vs-netstandard

Reply:
I can remove this blurb if everyone else agrees - I'd also need to remove it from create-and-publish-a-package-using-visual-studio-net-framework.md as well, to be consistent.

Reply:
I'm removing the blurb in both files.